From wfp5p@virginia.edu Wed Mar 25 08:05:44 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Wed, 25 Mar 2009 11:05:41 -0400
Subject: Staging: comedi: Remove boardtype typedef in addi-data
To: greg@kroah.com
Cc: abbotti@mev.co.uk, "fmhess@users.sourceforge.netds"@schleef.org
Message-ID: <20090325150541.13323.894.stgit@viridian.itc.Virginia.EDU>


typedef for boardtype removed and struct renamed to
addi_board

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/comedi/drivers/addi-data/addi_common.c |   10 +++++-----
 drivers/staging/comedi/drivers/addi-data/addi_common.h |    6 +++---
 2 files changed, 8 insertions(+), 8 deletions(-)

--- a/drivers/staging/comedi/drivers/addi-data/addi_common.c
+++ b/drivers/staging/comedi/drivers/addi-data/addi_common.c
@@ -73,7 +73,7 @@ You shoud also find the complete GPL in 
 //Update-0.7.57->0.7.68MODULE_LICENSE("GPL");
 
 #define devpriv ((addi_private *)dev->private)
-#define this_board ((boardtype *)dev->board_ptr)
+#define this_board ((struct addi_board *)dev->board_ptr)
 
 #if defined(CONFIG_APCI_1710) || defined(CONFIG_APCI_3200) || defined(CONFIG_APCI_3300)
 //BYTE b_SaveFPUReg [94];
@@ -219,7 +219,7 @@ static DEFINE_PCI_DEVICE_TABLE(addi_apci
 
 MODULE_DEVICE_TABLE(pci, addi_apci_tbl);
 
-static const boardtype boardtypes[] = {
+static const struct addi_board boardtypes[] = {
 #ifdef CONFIG_APCI_3120
 	{"apci3120",
 			APCI3120_BOARD_VENDOR_ID,
@@ -2525,7 +2525,7 @@ static const boardtype boardtypes[] = {
 #endif
 };
 
-#define n_boardtypes (sizeof(boardtypes)/sizeof(boardtype))
+#define n_boardtypes (sizeof(boardtypes)/sizeof(struct addi_board))
 
 struct comedi_driver driver_addi = {
       driver_name:"addi_common",
@@ -2534,7 +2534,7 @@ struct comedi_driver driver_addi = {
       detach:i_ADDI_Detach,
       num_names:n_boardtypes,
       board_name:&boardtypes[0].pc_DriverName,
-      offset:sizeof(boardtype),
+      offset:sizeof(struct addi_board),
 };
 
 COMEDI_PCI_INITCLEANUP(driver_addi, addi_apci_tbl);
@@ -2662,7 +2662,7 @@ static int i_ADDI_Attach(struct comedi_d
 		it->options[2]);
 	dev->irq = irq;
 
-	// Read eepeom and fill boardtype Structure
+	// Read eepeom and fill addi_board Structure
 
 	if (this_board->i_PCIEeprom) {
 		printk("\nPCI Eeprom used");
--- a/drivers/staging/comedi/drivers/addi-data/addi_common.h
+++ b/drivers/staging/comedi/drivers/addi-data/addi_common.h
@@ -61,7 +61,7 @@
 /* Structures */
 
 /* structure for the boardtype */
-typedef struct {
+struct addi_board {
 	const char *pc_DriverName;	// driver name
 	int i_VendorId;		//PCI vendor a device ID of card
 	int i_DeviceId;
@@ -199,7 +199,7 @@ typedef struct {
 	int (*i_hwdr_WriteTTLIOChlOnOff)(struct comedi_device *dev,
 					 struct comedi_subdevice *s,
 					 struct comedi_insn *insn, unsigned int *data);
-} boardtype;
+};
 
 //MODULE INFO STRUCTURE
 
@@ -406,7 +406,7 @@ typedef struct {
 
 	/* Pointer to the current process */
 	struct task_struct *tsk_Current;
-	boardtype *ps_BoardInfo;
+	struct addi_board *ps_BoardInfo;
 
 	/* Hardware board infos for 1710 */
 	struct {
